extension/chrome/settings/modules/contacts.ts

```diff
@@ -109,26 +110,32 @@ Catch.try(async () => {
 
   const actionProcessBulkImportTextInput = async () => {
     try {
-      const value = String($('#bulk_import .input_pubkey').val());
+      const value = Str.normalize(String($('#bulk_import .input_pubkey').val())).trim();
       const normalizedLongid = KeyImportUi.normalizeLongId(value);
-      let pubkey: string;
+      let key: string;
```

Comment:
I like to call pubkeys explicitly either `pub` or `pubkey` instead of just `key`, for easier security reviews.

Private keys could be called `prv` for example.

Basically nothing should be called just `key` because that's ambiguous, but I know there are many such places that we have to refactor.

Reply:
Yes, maybe it's an incorrect name of the variable,

I wanted to name it just `key` because we don't know whether the key is public or private or even some other string. We don't know what users can enter :)

I will rename it.